crates/core/src/masp.rs

```diff
@@ -263,25 +263,25 @@ pub fn encode_reward_asset_types(
             native_token.clone(),
             NATIVE_MAX_DECIMAL_PLACES.into(),
             MaspDigitPos::Zero,
-            Some(MaspEpoch::zero()),
+            None,
```

Comment:
I guess we should also update the comment here above

Reply:
Fixed, thanks!